integrations/astra/src/haystack_integrations/components/retrievers/astra/retriever.py

```diff
@@ -74,6 +85,7 @@ def to_dict(self) -> Dict[str, Any]:
             self,
             filters=self.filters,
             top_k=self.top_k,
+            filter_policy=self.filter_policy.value if self.filter_policy else None,
```

Comment:
```suggestion
            filter_policy=self.filter_policy.value,
```

In case of null values, the `__init__` fails, so we can be pretty sure the `self.filter_policy is not None`